File: 7-kyu/convert-a-linked-list-to-a-string/python/solution.py
def stringify(node):
    if node == None:
        return "None"
    nxt = node.next
    ret = [node.data]
    while nxt != None:
        ret.append(nxt.data)
        nxt = nxt.next
    ret.append("None")
    return ' -> '.join(str(item) if type(item) == int else item for item in ret)

File: 7-kyu/convert-a-linked-list-to-a-string/javascript/solution.js
const stringify = (list) => {
  if (list === null) {
    return "null";
  }
  let nxt = list.next;
  let ret = [list.data];
  while (nxt !== null) {
    ret.push(nxt.data);
    nxt = nxt.next;
  }
  ret.push("null");
  return ret.join(" -> ")
};
